add.go

```diff
@@ -88,6 +88,11 @@ func getURL(src string, chown *idtools.IDPair, mountpoint, renameTarget string,
 		return err
 	}
 	defer response.Body.Close()
+
+	if response.StatusCode < 200 || response.StatusCode >= 400 {
```

Comment:
Can we use constants from the http package instead (https://pkg.go.dev/net/http#pkg-constants)?